notebooks/advanced_RAG.ipynb

```diff
@@ -0,0 +1,1101 @@
+{
```

Comment:
<p>We can skip the "What is RAG" section, as it is an advanced notebook, and readers are likely to be familiar with the basics. You can add a link to a simple notebook for beginners <a href="https://github.com/huggingface/cookbook/blob/main/notebooks/RAG_zephyr_langchain.ipynb" target="_blank">https://github.com/huggingface/cookbook/blob/main/notebooks/RAG_zephyr_langchain.ipynb</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Thank you! It's done.</p>